kcidb/mq/argparse.py

```diff
@@ -0,0 +1,170 @@
+"""KICDB argparse"""
```

Comment:
Could you expand this a bit? To something like "KCIDB argument parsing"? Applies to similar docstrings in other modules too. Also, typo in "KCIDB".